neo.UnitTests/SmartContract/Native/Tokens/UT_GasToken.cs

```diff
@@ -140,5 +140,40 @@ public void Check_BadScript()
 
             NativeContract.GAS.Invoke(engine).Should().BeFalse();
         }
+
+        [TestMethod]
+        public void TestGetSysFeeAmount1()
+        {
+            using (ApplicationEngine engine = NativeContract.GAS.TestCall("getSysFeeAmount", 2u))
+            {
+                engine.ResultStack.Peek().GetBigInteger().Should().Be(new BigInteger(0));
```

Comment:
Also, I like to test the type (BigInteger, ByteArray..), because maybe it is returned as another type than expected with the same value